R/tufish_ebook.R

```diff
@@ -0,0 +1,86 @@
+
```

Comment:
There are actually two versions of the tuftish_ebook.R file (with different spellings). I think this is the one that should be deleted but let me know.